headers/types/dungeon_mode/dungeon.h

```diff
@@ -1783,6 +1791,9 @@ struct dungeon {
     undefined field_0x2ca45;
     undefined field_0x2ca46;
     undefined field_0x2ca47;
+    // 0x2CA0C: Probably a char[] because strcpy is called on it; however, unable to determine
```

Comment:
Since we know that it's a char array but don't know the length, can we make this a `char[1]`?